sovrin_node/test/test_suspension.py

```diff
@@ -81,9 +87,12 @@ def testTrustAnchorSuspensionByTrustee(looper, anotherTrustee, anotherTrustAncho
 
 
 def testTrusteeSuspensionByTrustee(looper, trustee, trusteeWallet,
-                                   anotherTrustee):
-    _, trWallet = anotherTrustee
+                                   anotherTrustee, anotherSteward1):
+    trClient, trWallet = anotherTrustee
     suspendRole(looper, trustee, trusteeWallet, trWallet.defaultId)
+    _, sWallet = anotherSteward1
+    with pytest.raises(AssertionError):
```

Comment:
Shouldn't we make a separate test for this case? Something like blacklistedTrusteeCanNotBlacklistSteward